lib/datarockets_style/cop/rspec/prefer_before_over_setup.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module DatarocketsStyle
+  module Cop
+    module RSpec
+      # bad
+      # setup do
+      #   allow(post).to receive(:publish!)
+      # end
+      #
+      # good
+      # before do
+      #   allow(post).to receive(:publish!)
+      # end
```

Comment:
```suggestion
      # Checks that tests use `before` instead of RoR unit-test `setup` method (part of `rspec-rails` gem)
      # 
      # bad
      # 
      # setup do
      #   allow(post).to receive(:publish!)
      # end
      #
      # good
      #
      # before do
      #   allow(post).to receive(:publish!)
      # end
```